### ISSUE TITLE
More information for failures would be good

### ISSUE THREAD

**Issue by thesayyn** [bazelbuild/rules_swift#318]
When something goes wrong, it is not possible to know what we did wrong since there's no verbose logs printed on terminal or logs files.

I dug to see if theres any option for that but I couldn't see any.

Thank you

**Comment by thesayyn** [comment]
Just figured I was using the 11.1 version. I think the documentation should be updated

**Comment by thomasvl** [comment]
I'm not sure what the follow up means since there isn't much detail here.

Also, what sorta things are you talking about for failures?  The compiler should be giving source errors.  If it is something else, you'll like need to provide more details as to what you were seeing to see if it is something the rules can influence or if it is a more general bazel error where the rules have no real control over what is output.

**Comment by thesayyn** [comment]
It does not print the compilation errors in version 11.1. When I updated rules to 12.1, I started to see compilation errors.  That means it was in fixed newest versions.

I think the documentation needs an update because  I got installed version 11.1  by following the installation guide of documentation

Bazel:  0.27.0
Apple Swift version 5.0.1 (swiftlang-1001.0.82.4 clang-1001.0.46.5)
Target: x86_64-apple-darwin18.7.0
